pkg/s3select/input.go

```diff
@@ -314,7 +255,8 @@ func (reader *Input) Execute(writer io.Writer) error {
 				close(myRow)
 				return nil
 			} else if ok {
-				message := reader.writeRecordMessage(row.record, curBuf)
+				jsonString := row.record
```

Comment:
any reason to keep this variable? why not use `row.record` ?

Reply:
Removed